
class ErrorMessageException(Exception):
    """Raised when the error message encounter"""
    pass
